plugins/community/chrusty-jsonschema/v1.3.9/buf.plugin.yaml

```diff
@@ -2,5 +2,5 @@ version: v1
 name: buf.build/community/chrusty-jsonschema
 plugin_version: v1.3.9
 source_url: https://github.com/chrusty/protoc-gen-jsonschema
-description: Converts protobuf definitions into JSONSchemas, which can be used to dynamically validate JSON messages.
+description: Converts protobuf definitions into JSON Schemas, which can be used to dynamically validate JSON messages.
```

Comment:
Should "Schema" be lower-case?

Reply:
I was following examples here: https://json-schema.org/. The GitHub repo description also uses "JSON-Schema".